### PR TITLE
Example structure

### PR DESCRIPTION
**Pull request by Baltmann01**
After discussing with @jasdumas (#34), I decided to create a short list/overview in the General Issues and reference the most common errors in the Documentation Issues.

### CODE REVIEW

**Review by jasdumas — APPROVED** [review]
👍

File: website/general_issues.qmd [hunk 1]
```diff
@@ -6,4 +6,63 @@ format:
...
+  + Examples which are supposed to fail and throw an error
+  + try() shows the error message but won't halt the execution
+  
+* **if(requireNamespace("packagename")){}**
+  + Used for examples which need particular packages to run
+  + Those packages should be listed in the Suggests filed of the DESCRIPTION file
+  
+* **\\dontshow{}:**
+  + Treated similarly to unwrapped examples but are not shown to the user
+  + Great for setting and resetting options or code purely for tests
```
Comment (llrs) [inline website/general_issues.qmd:58]: This is a very common pain point for developers. Also to be sure that this are correct (Even if CRAN rarely test them developers receive messages and deadlines regarding donttest), sometimes package maintainers are confused about when to use each one. 
I would recommend to add one or more examples for each case.

File: website/general_issues.qmd [hunk 2]
```diff
@@ -6,4 +6,63 @@ format:
...
+
+# Structuring of Examples 
+
+As part their documentation most functions have example code in their .Rd-files marked by the `\examples{}` tag. 
+Examples serve two major purposes for packages on CRAN.
+
+1. Show users how to implement the function correctly and what to expect as outcome.
+2. Allow the CRAN team to perform test on the code.
+
+For several reasons, some examples cannot be run as tests. Therefore, CRAN has several wrappers to structure them. The following list gives a short overview of the most common ones and when to use them.
```
Comment (llrs) [inline website/general_issues.qmd:26]: Maybe also mention the vignettes that are longer than examples but are also tested/checked.

**Review by llrs — COMMENTED** [review]
The examples is often a pain point. I left a couple of comments.

File: website/general_issues.qmd (hunk 1 above)
Comment (Baltmann01) [reply website/general_issues.qmd:58]: Good point, I thought about this as well but left it out for the first draft. I will include more details/examples here in the future, maybe from actual packages, maybe made up ones.

**Pull request merged**